source/val/decoration.h

```diff
@@ -62,7 +62,9 @@ class Decoration {
   void set_struct_member_index(uint32_t index) { struct_member_index_ = index; }
   int struct_member_index() { return struct_member_index_; }
   SpvDecoration dec_type() { return dec_type_; }
+  SpvDecoration dec_type() const { return dec_type_; }
```

Comment:
yeah, I think so too. 👍
I'll make this change after the current PRs go through because making the change now would cause rebase conflicts.